cache.go

```diff
@@ -190,8 +190,8 @@ func (c *Cache) Freshen(res *Resource, keys ...string) error {
 }
 
 func hashKey(key string) string {
-	h := sha256.New()
-	io.WriteString(h, key)
+	h := fnv.New64a()
+	h.Write([]byte(key))
```

Comment:
I did in there : https://gist.github.com/hectorj/8ac959c071d44ec4d718#file-cache_key_hash_test-go-L19

Though I only included the fastest results in the gist, and I don't have the numbers right here right now (will get them once I'm home)
IIRC Write([]byte(key)) instead of WriteString does help a bit, but FNV stays faster than sha256